backend/app/services/ingestion_service.py

```
@@ -0,0 +1,402 @@
+            # 解析文档内容
+            candidates = await self._extract_candidates(doc, task)
+
+            # 保存候选素材
+            for candidate in candidates:
+                self.db.add(candidate)
+
+            task.total_candidates = len(candidates)
+            task.processing_log.append({
+                "time": datetime.now(timezone.utc).isoformat(),
+                "action": "extraction_complete",
+                "message": f"提取完成，共 {len(candidates)} 个候选素材",
```

Ruff (0.15.9)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)